packages/node/src/notifier.js

```diff
@@ -53,6 +53,22 @@ const Bugsnag = {
 
     const bugsnag = new Client(opts, schema, internalPlugins, { name, version, url })
 
+    Object.keys(Client.prototype).forEach((m) => {
+      if (/^_/.test(m)) return
```

Comment:
I'm not sure that this is right — wouldn't we want to proxy the call even if it's a "private" method?

Reply:
Yes, this needs to apply to all method calls so that it's consistently applied to the same client e.g. if a public method then calls a private method they need directing to the same place